src/bench/rollingbloom.cpp

```diff
@@ -5,24 +5,20 @@
 
 #include <bench/bench.h>
 #include <common/bloom.h>
+#include <crypto/common.h>
 
 static void RollingBloom(benchmark::Bench& bench)
 {
     CRollingBloomFilter filter(120000, 0.000001);
     std::vector<unsigned char> data(32);
+    unsigned char* dataPtr = &data[0];
```

Comment:
```suggestion
    unsigned char* dataPtr = data.data();
```

Shouldn't this use the C++11 member function?

Reply:
Yeah, resolve this with the new commit.